superset-frontend/package-lock.json

```diff
@@ -24712,9 +24712,9 @@
       "integrity": "sha1-hnrHTjhkGHsdPUfZlqeOxciDB3c="
     },
     "typescript": {
-      "version": "3.5.3",
-      "resolved": "https://registry.npmjs.org/typescript/-/typescript-3.5.3.tgz",
-      "integrity": "sha512-ACzBtm/PhXBDId6a6sDJfroT2pOWt/oOnk4/dElG5G33ZL776N3Y6/6bKZJBFpd+b05F3Ct9qDjMeJmRWtE2/g==",
+      "version": "3.8.2",
```

Comment:
your PR title says this upgrades to 3.7.5, but it looks like it's actually upgrading to 3.8.2. i assume this isn't a problem, but you might want to update the title

Reply:
Fixed. They must've released a new version since I started working on this.